### ISSUE TITLE
A few typos

### ISSUE THREAD

**Issue by aljazerzen** [abuseofnotation/category-theory-illustrated#38]
This is a great book. I really wish I had it back at the university where all was explained with symbols only.

Here is a few typos / broken links I found: 

- missing dot in [first paragraph here](https://abuseofnotation.github.io/category-theory-illustrated/01_set/#purely-functional-programming-languages).
- missing image with alt text "A category" [here](https://abuseofnotation.github.io/category-theory-illustrated/02_category/#defining-categories-again)
- missing images [here](https://abuseofnotation.github.io/category-theory-illustrated/02_category/#associativity-and-reductionism)
- missing images [here](https://abuseofnotation.github.io/category-theory-illustrated/03_monoid/#what-are-monoids)

**Comment by abuseofnotation** [comment]
Thanks a lot, all should be fixed now.
